packaging/homebrew/README.md

```
@@ -113,9 +113,94 @@ brew audit --strict --formula ./packaging/homebrew/apispec.rb
+  out with `no_autobump!` (`Library/Homebrew/formula.rb`), so apispec would be on
+  the autobump list automatically. Homebrew's scheduled job notices a new
+  upstream release and BrewTestBot opens the version-bump PR itself.
+- **Detection already works with no `livecheck` block** — the default strategy
+  reads tags from the GitHub URL. Checked against the candidate formula:
+  `brew livecheck` reports `apispec: 0.5.6 ==> 0.5.6`, i.e. it found the current
+  release and agreed with it. When 0.5.7 is tagged that becomes
+  `0.5.6 ==> 0.5.7`, and the bump PR follows.
+- **To push a release in yourself** — a bot outage, or wanting it in core the
+  same day — it is one command, not the fork sequence:
+
+  ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)